src/mmw/js/src/draw/utils.js

```diff
@@ -72,6 +72,13 @@ function placeMarker(map, drawOpts) {
     return defer.promise();
 }
 
+function createRwdMarkerIcon(iconName) {
```

Comment:
Tried this and it looks like binding the popup prevents the `addTo` method from getting called correctly for the initial clicked point (and the popups seem to get bound in a slightly different way for the resulting points. Going to leave this as is for now.